src/client/version/versionpb/version.proto

```diff
@@ -11,6 +12,11 @@ message Version {
   string additional = 4;
 }
 
+message ClusterInfo {
+  string id = 1 [(gogoproto.customname) = "ID"];
+}
+
 service API {
   rpc GetVersion(google.protobuf.Empty) returns (Version) {}
+  rpc GetClusterInfo(google.protobuf.Empty) returns (ClusterInfo) {}
```

Comment:
The name of this method is inconsistent with our other names. RPCs that return `XInfo` should be called `InspectX`. I also don't think this makes sense in the `Version` API, since it's not really providing versioning information. Let's relocate this to the admin API, getting info about the cluster seems like an administrative task to me.

Unrelated to this PR would you mind renaming the `GetVersion` method to just `Version`? That seems more consistent to me. All the other `GetX` commands return a stream of data.